python/social_bot/tasks.py

```diff
@@ -906,9 +906,9 @@ class PickAndPlace(Task):
     def __init__(self,
                  env,
                  max_steps,
-                 object_random_range=0.5,
-                 place_to_random_range=0.5,
-                 min_distance=0.25,
+                 object_random_range=0.6,
+                 place_to_random_range=0.6,
+                 min_distance=0.3,
```

Comment:
Sometimes cube is moved into the body of youbot if min_distance is 0.25. 0.3 is more safer.